### PR TITLE
[Proposal] Distribute ownership of packages owned by integrations

### PR DESCRIPTION
**Pull request by jsoriano**
The integrations team is now distributed along different teams. Several packages are still assigned to the integrations team and responsibilities have diluted. It is not clear what current teams are taking care of them, what teams should review changes and so on.

The proposal would be to explicitly distribute the ownership of these packages along the current teams.

Proposed changes can be summarized as:
* Service integrations owned by @elastic/obs-service-integrations.
* System integrations (`system`, `windows`, `linux`) owned by @elastic/elastic-agent-data-plane.
* Input integrations (`journald`, `log`) owned by @elastic/elastic-agent-data-plane.
* `nginx_ingress_controller` owned by @elastic/obs-cloudnative-monitoring.

Some grey areas:
* `stan`, `nats`, `etcd` are services that can be used anywhere, but they are quite related to cloud native, so maybe they should be owned by the cloud native team.
* I think that input packages make sense in the data plane team, but the truth is that these packages have been maintained so far by SEI, ecosystem and other integrations teams, so not sure what to do.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -129,11 +129,11 @@
 /packages/sophos @elastic/security-external-integrations
 /packages/spring_boot @elastic/obs-service-integrations
 /packages/squid @elastic/security-external-integrations
-/packages/stan @elastic/integrations
+/packages/stan @elastic/obs-service-integrations
 /packages/suricata @elastic/security-external-integrations
 /packages/symantec_endpoint @elastic/security-external-integrations
 /packages/synthetics @elastic/uptime
-/packages/system @elastic/integrations
+/packages/system @elastic/elastic-agent-data-plane
```
Comment (cmacknz) [inline .github/CODEOWNERS:136]: @flash1293 Can we add the you or the Lens team as a code owner of the Kibana sub-directory of the system package: https://github.com/elastic/integrations/tree/main/packages/system/kibana?

That should add you to reviews of the dashboards and visualizations.

Comment (flash1293) [reply .github/CODEOWNERS:136]: Sounds good, you can add `@elastic/kibana-vis-editors`

Comment (jsoriano) [reply .github/CODEOWNERS:136]: Added, thanks!

Comment (cmacknz) [reply .github/CODEOWNERS:136]: The system dashboard doesn't change all that often, which is maybe good and bad for reviewing it. 

Maybe there are others that could benefit from shared ownership/review that are important and change more frequently. Kubernetes maybe? 

I don't have a good sense of what dashboards are getting attention in this repository

Comment (flash1293) [reply .github/CODEOWNERS:136]: It seems like aws is changing frequently too

Comment (cmacknz) [reply .github/CODEOWNERS:136]: Let's add the vis-editors team as code owners of the kuberenetes and AWS kibana directories too then @jsoriano . We can change it later if it is not useful or too noisy.

Comment (jsoriano) [reply .github/CODEOWNERS:138]: Added :+1:

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -144,14 +147,14 @@
 /packages/ti_recordedfuture @elastic/security-external-integrations
 /packages/ti_threatq @elastic/security-external-integrations
 /packages/tomcat @elastic/security-external-integrations
-/packages/traefik @elastic/integrations
+/packages/traefik @elastic/obs-service-integrations
 /packages/udp @elastic/security-external-integrations
-/packages/vsphere @elastic/integrations
-/packages/windows @elastic/integrations
+/packages/vsphere @elastic/obs-service-integrations
+/packages/windows @elastic/elastic-agent-data-plane
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply .github/CODEOWNERS:153]: I have thought of the security external integrations team a "contributor" to the windows integration. There are parts that we don't normally touch like perfmon and service monitoring. So perhaps from a Github codeowners perspective we should list both teams (similar to what we did on the gcp package).

**Comment by rameshelastic** [comment]
@jsoriano  I don't have the historical context of how this was working earlier but Service integrations look ok in the proposal above.

**Review by ChrsMark — COMMENTED** [review]
I think `nginx-ingress-controller` , `nats` and `stan` should be owned by @elastic/obs-cloudnative-monitoring , since those are “cloud native”  projects. Not sure about `etcd`. @luna can make the final call on this.

**Review by cmacknz — APPROVED** [review]

**Comment by jsoriano** [comment]
@ChrsMark is it possible that you wanted to ping @mlunadia on https://github.com/elastic/integrations/pull/3202#pullrequestreview-957441666? :slightly_smiling_face:

**Pull request merged**